server/server_app.js

```diff
@@ -105,14 +105,16 @@ app.post('/api/signup', (req, res, next) => {
     if (err) throw new Error(err)
     if (user == false) return res.json(info)
     const token = utils.generateToken(user.id)
-    gmail.sendEmail(
-      user.email,
-      'Email Verification',
-      'Thank you for you registration to our service !\nPlease go to the following link to confirm your mail : http://localhost:8080/api/mail/verification?token=' +
-        token
-    ).catch((_error) => {
-      return res.status(401).send('Invalid e-mail address.')
-    })
+    gmail
+      .sendEmail(
+        user.email,
+        'Email Verification',
+        'Thank you for you registration to our service !\nPlease go to the following link to confirm your mail : http://localhost:8080/api/mail/verification?token=' +
```

Comment:
just a question, do we let localhost by default ? Or precise the IP (and maybe the port) from a constant variable ?

Reply:
For the moment i let localhost:8080 to make the link clickable more faster in development phase.
We will rebase it when we professionalize the application (Part two or last MVP modification)

Reply:
Perfect